airbrakes/data_handling/logger.py

```diff
@@ -87,7 +87,7 @@ def log(
         self,
         state: str,
         extension: float,
-        imu_data_packets: deque[IMUDataPacket],
+        imu_data_packets: deque[EstimatedDataPacket | RawDataPacket],
```

Comment:
why are we doing this? doesn't this kinda defeat the point of polymorphism or whatever

Reply:
I guess so you can remove the extra isinstance... still feels wrong but ig its fine

Reply:
> I guess so you can remove the extra isinstance... still feels wrong but ig its fine

no that's just a type hint change to make pylance happy. It's also a little more specific but I see why one wouldn't like it. I'm fine with reverting if you wanna

Reply:
okay I guess not a super big deal but if you remember to, revert it